Cloud Application Manager/Automating Deployments/google-deployment-manager.md

```diff
@@ -11,8 +11,9 @@
 - [Overview](#overview)
 - [Audience](#audience)
 - [Prerequisites](#prerequisites)
-- [Getting started with Google Deployment Templates on Cloud Application Manager](#getting-started-with-google-deployment-templates-on-cloud-application-manager)
+- [Getting started with Google Deployment Templates](#getting-started-with-google-deployment-templates)
```

Comment:
Please, leave it as it was (it's best for search engine scoring to include the keywords several times)